src/tr1/game/gameflow.c

```diff
@@ -563,7 +567,11 @@ static bool M_LoadObjectNames(
         { "pickup1", "O_PICKUP_OPTION_1" },
         { "pickup2", "O_PICKUP_ITEM_2" },
         { "pickup2", "O_PICKUP_OPTION_2" },
+        { "leadbar", "O_LEADBAR_ITEM" },
+        { "leadbar", "O_LEADBAR_OPTION" },
+        { "scion", "O_SCION_OPTION" },
         { NULL, NULL },
+        // clang-format on
```

Comment:
Note: These guards are only useful if we do something against the autoformatter, such as aligning the items in columns.

Reply:
The autoformatter was putting this into columns after I added the extra entries so I added the guards - I assume there is a max line trigger for it?